actions/class.RestQtiTests.php

```diff
@@ -136,17 +136,16 @@ public function import()
         }
     }
 
-    protected function getItemClassUri(): ?string
+    /**
+     * @throws common_exception_RestApi
+     */
+    protected function getItemClassUri(): string
     {
-        $itemClassUri = $this->getPostParameter(self::ITEM_CLASS_URI);
-        $subclassLabel = $this->getSubclassLabel();
```

Comment:
We need to remove the constant https://github.com/oat-sa/extension-tao-testqti/pull/2495/files#diff-a24e0b70b24272218693b175599ec6df26afcadffda6a996e5a3a98e05d4a10cL49

Reply:
Done: https://github.com/oat-sa/extension-tao-testqti/pull/2495/commits/a52922f835e87ad828fa52796798b9d4b2610967

Thanks!